SimTransport/PPSProtonTransport/interface/BaseProtonTransport.h

```diff
@@ -41,37 +41,36 @@ class BaseProtonTransport {
   enum class TransportMode { HECTOR, TOTEM, OPTICALFUNCTIONS };
   TransportMode MODE;
 
-  int NEvent;
-  CLHEP::HepRandomEngine* engine_;
+  CLHEP::HepRandomEngine* engine_{nullptr};
   std::vector<LHCTransportLink> m_CorrespondenceMap;
   std::map<unsigned int, TLorentzVector> m_beamPart;
   std::map<unsigned int, double> m_xAtTrPoint;
   std::map<unsigned int, double> m_yAtTrPoint;
 
   bool verbosity_;
-  bool bApplyZShift;
+  bool bApplyZShift_;
   bool useBeamPositionFromLHCInfo_;
   bool produceHitsRelativeToBeam_;
 
-  std::string beam1Filename_;
-  std::string beam2Filename_;
+  std::string beam1Filename_{""};
+  std::string beam2Filename_{""};
 
-  double fPPSRegionStart_45;
-  double fPPSRegionStart_56;
-  double fCrossingAngleX_45;
-  double fCrossingAngleX_56;
-  double fCrossingAngleY_45;
-  double fCrossingAngleY_56;
+  double fPPSRegionStart_45_;
+  double fPPSRegionStart_56_;
+  double fCrossingAngleX_45_{0.0};
+  double fCrossingAngleX_56_{0.0};
+  double fCrossingAngleY_45_{0.0};
+  double fCrossingAngleY_56_{0.0};
 
   double beamMomentum_;
   double beamEnergy_;
   double etaCut_;
   double momentumCut_;
 
-  double m_sigmaSTX;
-  double m_sigmaSTY;
-  double m_sigmaSX;
-  double m_sigmaSY;
-  double m_sig_E;
+  double m_sigmaSTX{0.0};
```

Comment:
As above: please initialize these `m_SigmaS*` and `m_sig_E` in the constructor of `BaseProtonTransport` (via `iCongif.getParameter` etc.) or move these class members to the derived classes which actually use them